deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml

```diff
@@ -84,7 +84,7 @@ spec:
             - name: socket-dir
               mountPath: /csi
         - name: csi-snapshotter
-          image: quay.io/k8scsi/csi-snapshotter:v2.0.1
+          image: quay.io/k8scsi/csi-snapshotter:v2.1.1
```

Comment:
In the master, can you just update the image to canary?
In the stable branches, can up submit separate PRs to update image to specific versions?

Reply:
@xing-yang i have addressed the changes.PTAL